src/Symfony/Component/AssetMapper/Factory/MappedAssetFactory.php

```diff
@@ -73,6 +74,18 @@ public function createMappedAsset(string $logicalPath, string $sourcePath): ?Map
         return $this->assetsCache[$logicalPath];
     }
 
+    /**
+     * Returns an SRI integrity hash for the given asset.
+     */
+    private function getIntegrity(MappedAsset $asset, ?string $content): string
+    {
+        if (null !== $content) {
+            return 'sha384-'.base64_encode(hash('sha384', $content, true));
+        }
+
+        return 'sha384-'.base64_encode(hash_file('sha384', $asset->sourcePath, true));
```

Comment:
Could be simplified to: 
```suggestion
        $hash = $content !== null ? hash('sha384', $content, true) : hash_file('sha384', $asset->sourcePath, true);

        return 'sha384-'.base64_encode($hash);
```

Reply:
Not sure if adding a var is worth it here. This code is not in the hotpath, so i'd maybe vote for code readability. 

wdyt ?